docs/config/README.md

```diff
@@ -105,6 +106,7 @@ prompt_order = [
     "ruby",
     "rust",
     "terraform",
+    "terraform",
```

Comment:
It looks like this line was duplicated. 🤔 
```suggestion
```

Reply:
Removed. 👍